tests/functional/event_handler/_pydantic/test_openapi_params.py

```diff
@@ -172,6 +172,42 @@ def handler() -> Response[Annotated[str, Body(title="Response title")]]:
     assert response.schema_.type == "string"
 
 
+def test_openapi_with_tuple_returns():
+    app = APIGatewayRestResolver()
+
+    @app.get("/")
+    def handler() -> tuple[str, int]:
```

Comment:
Can you pls import `Tuple` from typing? I need to migrate our tests to adopt the new type annotation, but we have some issues for now.

Reply:
Done.